geonotebook/kernel.py

```diff
@@ -404,8 +407,16 @@ def handle_comm_open(self, comm, msg):
 
         # THis should be handled in a callback that is fired off
         # When set protocol etc is complete.
-        self.geonotebook.add_layer(None, name="osm_base", layer_type="osm",
-                                   vis_url="http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png")
+        self.geonotebook.add_layer(
+            None, name="osm_base", layer_type="osm",
+            vis_url="http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+            system_layer=True)
+
+        self.geonotebook.add_layer(
```

Comment:
I can't see any immediate problems with this, but isn't there a bit of a code smell around the fact that an annotation layer exists even when there are no annotations?

The alternative being to create an annotation layer when the first annotation is drawn, I'm not sure if that's cleaner.